pkgs/applications/misc/jotta-cli/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchzip }:
+
+let
+  arch = "amd64";
+in
+stdenv.mkDerivation rec {
+  pname = "jotta-cli";
+  version = "0.6.18626";
+  src =
+    fetchzip {
+      url = "https://repo.jotta.us/archives/linux/${arch}/jotta-cli-${version}_linux_${arch}.tar.gz";
+      sha256 = "0v9bw0f2mcvmzp7v8gs6q4p1q54rflqnbjv5sw7h1kyfwznmflzj";
+      stripRoot = false;
+    };
+
+  installPhase = ''
+    install -D usr/bin/jotta-cli usr/bin/jottad -t $out/bin/
+  '';
+
+  postInstall = ''
+    mkdir -p $out/share/bash-completion/completions
+    $out/bin/jotta-cli completion > $out/share/bash-completion/completions/jotta-cli.bash
```

Comment:
Was confused for a bit there as the output doesn't include `share/`. That happens because `postInstall` isn't run automatically when overriding `installPhase`.

Anyway, we can't run `jotta-cli` before we've patched the interpreter in `postFixup`. Probably best to just create the directory in `installPhase` and run the script after patching.

Reply:
Ah, completely missed that, poor testing on my side.